packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -74,10 +82,13 @@ export const makeContract = harden(zcf => {
     const internalTicketSupplyOfferHook = offerHandle =>
       (internalTicketSupplyHandle = offerHandle);
 
-    const contractSelfInvite = zcf.makeInvitation(
-      internalTicketSupplyOfferHook,
-    );
-    // the contract creates an offer {give: tickets, want: nothing} with the tickets
+    const contractSelfInvite = inviteAnOffer({
+      offerHook: internalTicketSupplyOfferHook,
+      inviteDesc: 'contract self invite',
+      expected: { give: { Ticket: null } },
```

Comment:
What's the reasoning behind sometimes specifying the expected value, and often not?

Reply:
My preference would be to check `expected` more places. However, in this PR I was mostly trying to preserve the semantics of the previous code, including where it checks `expected`.